### PR TITLE
add support on Debian likes

### PR DESCRIPTION
**Pull request by dirks**

### CODE REVIEW

File: vars/archlinux.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (wookietreiber) [inline vars/archlinux.yml:1]: Please add an empty line after all `---`. Unfortunately, I haven't finished [adding this to yamllint](https://github.com/adrienverge/yamllint/pull/393), but that's one check that's missing :smile:

File: vars/archlinux.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+locale_conf_path: /etc/locale.conf
+...
```
Comment (wookietreiber) [inline vars/archlinux.yml:5]: Please add an empty line before all `...`. Unfortunately, I haven't finished [adding this to yamllint](https://github.com/adrienverge/yamllint/pull/393), but that's one check that's missing :smile:

File: vars/archlinux.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+locale_conf_path: /etc/locale.conf
```
Comment (wookietreiber) [inline vars/archlinux.yml]: Please use the `__` double underscore prefix for internal variables, i.e. `__locale_conf_path`.

**Review by wookietreiber — CHANGES_REQUESTED** [review]

File: vars/archlinux.yml (hunk 3 above)
Comment (dirks) [reply vars/archlinux.yml]: I guess that should be consistent in the role and variables such as `locale_localectl_cmd` which is registered in `tasks/main.yml` should follow the same pattern?

I would propose to change the internal variable pattern in a commit separate from this pull request. I would rebase this pull request Agreed?

**Review by wookietreiber — APPROVED** [review]

**Pull request merged**